fido-key-manager/src/main.rs

```diff
@@ -69,8 +69,8 @@ pub struct RenameFingerprintOpt {
 
 #[derive(Debug, Args)]
 pub struct RemoveFingerprintOpt {
-    /// The template ID
-    #[clap(min_values(1), required(true))]
+    /// The template ID(s) to remove
```

Comment:
Does this actualy assert that 1 or more ids are provided.

Reply:
Yes:

```
> .\target\debug\fido-key-manager.exe remove-fingerprint
error: the following required arguments were not provided:
  <ID>...

Usage: fido-key-manager.exe remove-fingerprint <ID>...

For more information, try '--help'.
```